sysbox-in-docker/sysbox

```diff
@@ -185,10 +185,10 @@ function docker_setup() {
             "path": "/usr/local/sbin/sysbox-runc"
         }
     },
-    "bip": "172.24.0.1/16",
+    "bip": "172.25.0.1/16",
```

Comment:
Yes, good catch, there could be network overlaps if sysbox is also installed in the host.

Reply:
Correct.